dev.zelikos.rollit.json

```diff
@@ -0,0 +1,49 @@
+{
+    "app-id" : "dev.zelikos.rollit",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "rollit",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--env=RUST_BACKTRACE=1",
+        "--env=RUST_LOG=rollit=debug"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
+        ]
```

Comment:
this is not allowed. build MUST be offline.

Either use https://github.com/flatpak/flatpak-builder-tools to generate the list from the cargo lock, or use the tarball since it looks like the project use the standard meson template.